packages/playground/src/components/webhooks/create-webhooks-form-fields.tsx

```diff
@@ -28,67 +28,67 @@ export const WebhookToggleField: React.FC<FieldProps> = ({ register, watch, setV
+    <Input id="name" {...register!('identifier')} placeholder="Name your webhook" autoFocus disabled={disabled} />
+    {errors!.identifier && (
+      <FormFieldSet.Message theme={MessageTheme.ERROR}>{errors!.identifier.message?.toString()}</FormFieldSet.Message>
     )}
   </FormFieldSet.ControlGroup>
 )
 
-export const WebhookDescriptionField: React.FC<FieldProps> = ({ register, errors }) => (
+export const WebhookDescriptionField: React.FC<WebhookFormFieldProps> = ({ register, errors }) => (
   <FormFieldSet.ControlGroup>
-    <FormFieldSet.Label htmlFor="description" required>
-      Description
-    </FormFieldSet.Label>
+    <FormFieldSet.Label htmlFor="description">Description</FormFieldSet.Label>
     <Textarea id="description" {...register!('description')} placeholder="Enter a description of this rule..." />
     {errors!.description && (
       <FormFieldSet.Message theme={MessageTheme.ERROR}>{errors!.description.message?.toString()}</FormFieldSet.Message>
     )}
   </FormFieldSet.ControlGroup>
 )
 
-export const WebhookPayloadUrlField: React.FC<FieldProps> = ({ register, errors }) => (
+export const WebhookPayloadUrlField: React.FC<WebhookFormFieldProps> = ({ register, errors }) => (
   <FormFieldSet.ControlGroup>
     <FormFieldSet.Label htmlFor="payloadUrl" required>
       Payload URL
     </FormFieldSet.Label>
-    <Input id="payloadUrl" {...register!('payloadUrl')} placeholder="https://example.com/harness" />
-    {errors!.payloadUrl && (
-      <FormFieldSet.Message theme={MessageTheme.ERROR}>{errors!.payloadUrl.message?.toString()}</FormFieldSet.Message>
+    <Input id="payloadUrl" {...register!('url')} placeholder="https://example.com/harness" />
+    {errors!.url && (
+      <FormFieldSet.Message theme={MessageTheme.ERROR}>{errors!.url.message?.toString()}</FormFieldSet.Message>
     )}
   </FormFieldSet.ControlGroup>
 )
 
-export const WebhookSecretField: React.FC<FieldProps> = ({ register, errors }) => (
+export const WebhookSecretField: React.FC<WebhookFormFieldProps> = ({ register, errors }) => (
   <FormFieldSet.ControlGroup>
-    <FormFieldSet.Label htmlFor="secret" required>
-      Secret
-    </FormFieldSet.Label>
+    <FormFieldSet.Label htmlFor="secret">Secret</FormFieldSet.Label>
     <Input id="secret" {...register!('secret')} type="password" />
     {errors!.secret && (
       <FormFieldSet.Message theme={MessageTheme.ERROR}>{errors!.secret.message?.toString()}</FormFieldSet.Message>
     )}
   </FormFieldSet.ControlGroup>
 )
 
-export const WebhookSSLVerificationField: React.FC<FieldProps> = ({ watch, setValue }) => {
-  const sslVerificationValue = watch!('sslVerification')
+export const WebhookSSLVerificationField: React.FC<WebhookFormFieldProps> = ({ watch, setValue }) => {
+  const sslVerificationValue = watch!('insecure')
   const handleAccessChange = (value: string) => {
-    setValue!('sslVerification', value as '1' | '2')
+    setValue!('insecure', value as '1' | '2')
```

Comment:
nit: Anything that we can do improve readability of this? `value as '1' | '2'`

Reply:
removed. Added in schema.